scripts/ci/pr-bot/processNewPrs.ts

```diff
@@ -45,15 +45,24 @@ import { CheckStatus } from "./shared/checks";
  * (in which case that's all we need to do).
  */
 function needsProcessed(pull: any, prState: typeof Pr): boolean {
-  if (!pull.labels.find((label) => label.name.toLowerCase() === "go")) {
+  if (
```

Comment:
re item 6) above : for my education, what does it mean to have notifications stopped?
item 7 is out of sync now.

Reply:
Anytime a reviewer manually requests review (e.g. `R: @user`), the bot turns itself off (and comments that its doing so. For example - https://github.com/apache/beam/pull/21852#issuecomment-1154750727

Fixed the comment